documentation/airport-file-standards.md

```diff
@@ -0,0 +1,89 @@
+
+### All SIDs & STARs
+All procedures for a given airport must be included in the pull request. Not all have to be _used_ for anything, but (to the extent possible) all SIDs and STARs must be present in the file.
+
+### Accurate traffic level (and info to back it up)
+The traffic level must be commensurate with real-world levels, with the arrival and departure rates. Airports can be very quiet during some hours, and very busy during others-- therefore, the traffic included in the airport file shall be:
+- not less than `yearly operations / 8760` (hourly equivalent of yearly rate)
+- not more than `3x yearly operations / 8760` (hourly equivalent of yearly rate)
+
+A reference link shall be provided, validating the yearly traffic count, so the reviewer may confirm that the traffic level is appropriate.
+
+### Base video map
+A "video map" must be provided with all airports. For most US airports, these are easily found by checking with the appropriate ARTCC within VATUSA on VATSIM. In many other countries, however, these are not available to the public. If the originals cannot be obtained, an appropriate and well-thought-out video map must be created. qGIS is the best place to do this.
+
+### Spacing
+To maintain uniformity in all airport files, only single spaces should be used.
+
+The _single_ exception to this rule is in the `fixes` section, where all five-and-fewer-character fixes should have their colons aligned, as in the accepted example below:
+```
+"CUSBU": ["N47d29.91m0", "W122d19.05m0"],
+"CVO"  : ["N44d29.97m0", "W123d17.62m0"],
+"GRIFY": ["N47d46.15m0", "W122d24.12m0"],
+"HAROB": ["N47d14.60m0", "W123d02.45m0"],
+"HAWKZ": ["N46d47.08m0", "W122d44.34m0"],
+"HETHR": ["N47d33.54m0", "W122d08.70m0"],
+"HISKU": ["N44d30.00m0", "W122d56.65m0"],
+"HQM"  : ["N46d56.82m0", "W124d08.96m0"],
+"LACEE": ["N47d02.82m0", "W122d48.35m0"],
+"LIINE": ["N46d50.65m0", "W122d40.11m0"],
+"LKV"  : ["N42d29.57m0", "W120d30.42m0"],
+"MARNR": ["N48d02.33m0", "W122d50.99m0"]
+```
+
+For custom RNAV fixes, such as `_RWY22L05DME`, these shall use single spaces.
+
+### Bracket Newlines
+To maintain uniformity in all airport files, please follow the same newline conventions as the example file shown in the [airport format](./airport-format.html) document.
+
+### Key order
+To maintain uniformity in all airport files, all keys used in the file must be included in the same order as they appear in the [airport format](./airport-format.html) document.
+- All fixes must be sorted alphabetically.
+- The first object within the `airspace` array must represent the airspace's _outer perimeter_, which encompasses all other airspace sections.
+- All runways should be sorted numerically (then L-C-R) by the primary end
+- All SIDs/STARs must be sorted alphabetically by ID
+- Spawn patterns should have _departure_ patterns, then _arrival_ patterns, then any _overflight_ patterns.
```

Comment:
`spawnPatterns` have their own key/value enforcement that could be touched on here.  at the very least we should link to that document (or stub it in, cause it moves in #184 )